linux_os/guide/system/permissions/mounting/kernel_module_freevxfs_disabled/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhcos4,rhel7,rhel8,rhel9,rhv4,sle15,ubuntu1804,ubuntu2004,ubuntu2204
+prodtype: fedora,ol7,ol8,rhcos4,rhel7,rhel8,rhel9,rhv4,sle15,ubuntu1804,ubuntu2004
```

Comment:
Removing a product from `prodtype` might cause issues for users who used this rule in any tailored file. With this removal, the rule will no longer be present in the Datastream.
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#agreements

Considering this, is the removal indeed desired?

Reply:
yes it is desired, that rule is not part of CIS for 22.04
unfortunately that was added as someone ported CIS for 20.04 to build on 22.04 before we had a proper CIS for 22.04. That won't happen in the future.

Reply:
Ok. Thanks for confirming.